src/hooks/useDrafts/useDrafts.tsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react'
+import { addOrUpdateDraft, clearDrafts, getDraft, getDrafts, removeDraft } from './utils'
+
+type CommonDraftProps = { id: string; recentlyUpdatedAt: string }
```

Comment:
My guess it the idea is to allow to store any kind of draft here, not just a video draft, right? If so, we probably should add some field here that will tell us what kind of drafts this is